src/network/clientpackethandler.cpp

```diff
@@ -120,7 +121,12 @@ void Client::handleCommand_AuthAccept(NetworkPacket* pkt)
 					<< m_recommended_send_interval<<std::endl;
 
 	// Reply to server
-	NetworkPacket resp_pkt(TOSERVER_INIT2, 0);
+	std::string lang = gettext("LANG_CODE");
```

Comment:
What's the problem with using `TOSERVER_INIT2` to send the language wanted? It is the packet the server replies to with media announcement, so since we want it to depend on that language and we don't need it elsewhere, I don't see why it isn't the best place for it.